CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,176 @@
+- Help maintain project quality control via [code reviews] on PRs
+  - Focus on code quality and correctness, including testing and factoring
+  - May also review for more holistic issues, but not a requirement
+- Be responsive to review requests
+- May be assigned PRs to review if in area of expertise
+- Assigned test bugs related to the project of expertise
+
+#### Privileges
+
+- Same as for Community Members
+- Triager status may be a precondition to accepting large code contributions
+
+#### Promotion process
+
+- Sponsored by a maintainer
+  - With no objections from other maintainers
+  - Done through issue or PR to update the CODEOWNERS file
+- May self-nominate or be nominated by a maintainer
+
+### Maintainers
+
+Maintainers are responsible for the project's overall health.
+They are the only ones who can approve and merge code contributions.
+While triage and code review is focused on code quality and correctness,
+approval is focused on holistic acceptance of a contribution including:
+
+- backwards/forwards compatibility
+- adherence to API and flag conventions
+- subtle performance and correctness issues
+- interactions with other parts of the system
+- consistency between code and documentation
+
+**Defined by:** "Maintain" permissions in the project and an entry in its CODEOWNERS file
+
+#### Pre-requisites
+
+- Triager for at least 3 months
+- Reviewed at least 10 substantial PRs to the codebase
+- Reviewed or got at least 30 PRs merged to the codebase
+
+#### Responsibilities
+
+- Demonstrate sound technical judgment
+- Maintain project quality control via code reviews
+  - Focus on holistic acceptance of contribution
+- Be responsive to review requests
+- Mentor contributors and triagers
+- Approve and merge code contributions as appropriate
+- Participate in OpenSSF or Scorecard-specific community meetings, if possible
+- Hosting Scorecard-specific community meetings, if possible and comfortable
+
+#### Privileges
+
+- Same as for Triager
+- Maintainer status may be a precondition to accepting especially large code contributions
+
+#### Promotion process
+- Sponsored by a maintainer
+  - With no objections from other maintainers
+  - Done through PR to update the CODEOWNERS file
+- May self-nominate or be nominated by a maintainer
+
+## Inactive members
+A core principle in maintaining a healthy community is encouraging active participation.
+It is inevitable that a contributor's focus will change over time
+and there is no expectation they'll actively contribute forever.
+
+Any contributor at any level described above may write an issue (or PR, if CODEOWNER changes are necessary)
+asking to step down to a lighter-weight tier or to depart the project entirely.
+The change in responsibilities will be understood to take immediate effect,
+regardless of whether the issue/PR has been acknowledged or merged.
+
+However, should a Triager or above be deemed inactive for a significant period, any
```

Comment:
curious if this should be discussed prior to sending a PR

Reply:
I don't see an update about consulting with the team

Reply:
`Such requests will hopefully come after thoughtful conversations with the rest of the team and with sufficient forewarning for the others to prepare.`

Is this not what you meant?

Reply:
`Such requests will hopefully come after thoughtful conversations with the rest of the team and with sufficient forewarning for the others to prepare`. Is this not what you meant? (it doesn't appear in the diff above).

Reply:
It is! Sorry I did not se the update at the time I read :)